test_psa_target.py

```diff
@@ -127,6 +127,33 @@ def _build_tfm(args, config, suite=None):
         sys.exit(1)
 
 
+def _erase_flash_storage(args):
+    """
+    Creates a target specific binary which has its ITS erased
+    :param args: Command-line arguments
+    """
+    mbed_os_dir = join(ROOT, "BUILD", args.mcu, TC_DICT.get(args.toolchain))
+    cmd = []
+
+    if args.mcu == "ARM_MUSCA_B1":
+        cmd = [
+            "srec_cat",
+            "mbed-os-tf-m-regression-tests.bin",
+            "-Binary",
+            "-offset",
+            "0xA000000",
+            "-fill",
+            "0xFF",
+            "0xA1F0000",
+            "0XA1FC000",
+            "-o",
+            "mbed-os-tf-m-regression-tests-reset-flash.hex",
+            "-Intel",
+        ]
+
+    run_cmd_output_realtime(cmd, mbed_os_dir)
```

Comment:
Does this return an error like `run_cmd_and_return()` does? If so, we should handle the error. Disregard my comment here if `run_cmd_output_realtime()` throws an exception on failure.